package_test.go

```diff
@@ -32,7 +32,7 @@ var setupSSH string
 //go:embed tests/docker-entrypoint.sh
 var entrypointCmd string
 
-const entrypointVersion = "5ebaa181f866e32e59e37face813cf25b74e8911"
+const entrypointVersion = "892ed9a42ceb5f9a9c7198adfc316da64a573274"
```

Comment:
Was there a relevant change in the entrypoint? Or is this just to keep it at the latest version?

Reply:
The change removes the implicit SSH socket forward for docker.sock which didn't work for how we are using it and caused errors.
https://github.com/cpuguy83/qemu-micro-env/compare/5ebaa181f866e32e59e37face813cf25b74e8911...892ed9a42ceb5f9a9c7198adfc316da64a573274